transport/src/main/java/io/netty/channel/DefaultFileRegion.java

```diff
@@ -120,6 +120,11 @@ public long transfered() {
     }
```

Comment:
I guess we should also add a `@Deprecated` annotation on the implementations of the `transfered` method too